.github/settings.yml

```
@@ -26,4 +26,15 @@ collaborators:
   
   - username: mkorbi
     permission: admin
+    
```

I suggest non-cncf staff for now have `maintain` access (see https://probot.github.io/apps/settings/ for all settings).

Also, for clarity, suggesting alphabetical order and note that these people are bootstrapped for now beauze formalizing TAG reps has not yet taken place.

```suggestion
  # maintain: Bootstrap group volunteers prior to governance voting
  # Alphabetically

  - username: catblade
    permission: maintain

  - username: cdeliaRH
    permission: maintain

  - username: nikimanoledaki
    permission: maintain

  - username: mkorbi
    permission: maintain
    
  - username: scottrigby
    permission: maintain
    
```